zebra/zebra_l2.c

```diff
@@ -50,7 +50,8 @@
 /* static function declarations */
 
 /* Private functions */
-static void map_slaves_to_bridge(struct interface *br_if, int link)
+static void map_slaves_to_bridge(struct interface *br_if, int link,
+				 bool update_slave, int chgflags)
```

Comment:
chgflags should be uint16_t, or uint32_t depending on the need.

Reply:
ok, I will use uint8_t